tests/test_bmidelta.py

```diff
@@ -508,18 +498,18 @@ def test_update_until(self, tmp_path):
         delta.update_until(2)
         assert delta._delta._time == 2.
 
-    @pytest.mark.xfail(raises=ValueError, strict=True,
-                       reason="Recalcualtion of Np_water on timestep change causes break. Upstream error.")
+    @pytest.mark.xfail(raises=ValueError, strict=False,
+                       reason="Recalculation of Np_water on timestep change causes break. Upstream error.")
     def test_update_frac(self, tmp_path):
         filename = 'user_parameters.yaml'
         p, f = create_temporary_file(tmp_path, filename)
         write_parameter_to_file(f, 'model_output__out_dir', tmp_path / 'out_dir')
         f.close()
         delta = BmiDelta()
         delta.initialize(p)
-        with pytest.warns(UserWarning):
-            delta.update_frac(0.2)
-        assert delta._delta.time_step == 1.0
+        # with pytest.warns(UserWarning):
+        delta.update_frac(0.2)
+        assert delta._delta.time_step == 25000.0
```

Comment:
so what is happening with this test now? does the test just pass -- if so, we should just remove the decorator.

Reply:
This does pass, so I will go in and remove the decorator. I think the default parameters have just changed since the BMI wrapper was first written. We used to have that tiny domain which had issues with stability and some of the model parameters, I think now that the default parameters are for a more regular model configuration, some of these things that previously broke now work. But I'm not exactly sure why

Reply:
Okay, that's a good enough explanation for me :+1: thanks for getting to the bottom of it!